frontend/src/metabase/containers/AddToDashSelectDashModal/AddToDashSelectDashModal.unit.spec.tsx

```diff
@@ -573,6 +659,107 @@ describe("AddToDashSelectDashModal", () => {
               }),
             ).toBeInTheDocument();
           });
+
+          describe('when "Create a new dashboard" option is clicked', () => {
+            beforeEach(async () => {
+              // "Create a new dashboard" option only renders when opening personal collections
+              userEvent.click(
+                screen.getByRole("heading", {
+                  name: PERSONAL_COLLECTION.name,
+                }),
+              );
+              userEvent.click(
+                screen.getByRole("heading", {
+                  name: "Create a new dashboard",
+                }),
+              );
+            });
+
+            it("should render only personal collections", async () => {
+              // Create a new dashboard modal
+              expect(
+                screen.getByRole("heading", { name: "New dashboard" }),
+              ).toBeInTheDocument();
+              userEvent.click(screen.getByTestId("select-button"));
+
+              const popover = screen.getByRole("tooltip");
+              expect(popover).toBeInTheDocument();
+              expect(
+                await within(popover).findByRole("heading", {
+                  name: PERSONAL_COLLECTION.name,
+                }),
+              ).toBeInTheDocument();
+              expect(
+                within(popover).queryByRole("heading", {
+                  name: "Our analytics",
+                }),
+              ).not.toBeInTheDocument();
+              expect(
+                within(popover).queryByRole("heading", {
+                  name: COLLECTION.name,
+                }),
+              ).not.toBeInTheDocument();
+            });
+
+            it('should not render "New collection" option', async () => {
+              // Create a new dashboard modal
+              expect(
+                screen.getByRole("heading", { name: "New dashboard" }),
+              ).toBeInTheDocument();
+              userEvent.click(screen.getByTestId("select-button"));
+
+              const popover = screen.getByRole("tooltip");
+              expect(popover).toBeInTheDocument();
+              expect(
+                await within(popover).findByRole("heading", {
+                  name: PERSONAL_COLLECTION.name,
+                }),
+              ).toBeInTheDocument();
+              expect(
+                within(popover).queryByText("New collection"),
+              ).not.toBeInTheDocument();
+            });
+
+            describe('when "New collection" option is clicked', () => {
+              beforeEach(async () => {
+                // Create a new dashboard modal
+                userEvent.click(screen.getByTestId("select-button"));
```

Comment:
To navigate collections in a collection select, we need to click the arrow button.
![Screenshot 2023-11-06 at 10 23 10 PM](https://github.com/metabase/metabase/assets/1937582/0f9230d6-48af-434e-9075-41144a53047c)